.github/ISSUE_TEMPLATE/blank_template.md

```diff
@@ -0,0 +1,10 @@
+---
+name: Other (blank template)
```

Comment:
Not sure we need this?

Reply:
Do you mean the template as a whole or the "(blank template)" sentence?

Reply:
This probably fits a bunch of issues maintainers create. We could try and fit our own issues in the other categories, but I guess having this wouldn't hurt. No strong opinion though.

Reply:
Do we want to add an RFC (whatever RFC means)?

Reply:
C -> changes or comments :)